controllers/user.controller.js

```diff
@@ -1,9 +1,34 @@
 import User from "../models/user.model.js";
-import { fMsg,paginate} from "../utils/libby.js";
+import { fileURLToPath } from "url";
+import { fMsg, paginate } from "../utils/libby.js";
+import fs from "fs";
+import path from "path";
+import { deleteFile } from "../utils/libby.js";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
 
 export const updateUserInfo = async (req, res) => {
     try {
         const userId = req.user._id;
+        const userInfo = await User.findById(userId);
+        const username = req.body.userName;
```

Comment:
good job.. but i recommend to use about "username" to "userName"  consistently .. i know it does not effect anything in app logic but it will be in pain when we have to debug about userName problems in the future